pyaerocom/aeroval/helpers.py

```diff
@@ -130,3 +147,56 @@ def _get_min_max_year_periods(statistics_periods):
         if perstop > stopyr:
             stopyr = perstop
     return startyr, stopyr
+
+
+def make_dummy_model(obs_list: list, cfg) -> str:
+
+    # Sets up variable for the model register
+    tmpdir = const.LOCAL_TMP_DIR
+    const.add_data_search_dir(tmpdir)
+
+    model_id = "dummy_model"
+    outdir = os.path.join(tmpdir, f"{model_id}/renamed")
+
+    os.makedirs(outdir, exist_ok=True)
```

Comment:
not sure if `GriddedData.to_netcdf` accepts `pathlib.Path` objects but it might be cleaner to write this part with a Path object

``` python
    outdir = Path(tmpdir) / f"{model_id}/renamed"
    outdir.mkdir(parents=True, exist_ok=True)
```

Reply:
`GriddedData.to_netcdf` uses `os.path.join`, so I think it must be str. If we are going to change this, then it should be its own PR

Reply:
`os.path.join` accepts [path-like objects](https://docs.python.org/3/glossary.html#term-path-like-object) (see [docs](https://docs.python.org/3/library/os.path.html#os.path.join)),
which means that it will accept `pathlib.Path` objects

Reply:
@dulte ?